spec/concurrency_spec.rb

```diff
@@ -13,7 +13,7 @@ def write_thread(resource_id, content = "unchanging", &block)
       resource = storage.resource(resource_id.to_s)
 
       if block
-        block.call(resource)
+        yield(resource)
```

Comment:
You could try replacing `if block` with `if block_given?` (and remove the argument `, &block`)

Reply:
bleh totally missed this comment